### PR TITLE
[ERDE-1917] getRun fields

### PR DESCRIPTION
**Pull request by sophos-daniels**
Update api client `getRun` has to explicitly request fields from API now.

### CODE REVIEW

File: jest.config.ts [hunk 1]
```diff
@@ -5,7 +5,8 @@ const config: Config.InitialOptions = {
   testMatch: ['**/?(*.)+(test).+(ts|js)'],
   transform: {
     '^.+\\.ts$': 'ts-jest'
-  }
+  },
+  testEnvironment: 'node'
```
Comment (sophos-carlos) [inline jest.config.ts:9]: `node` is the default env, but doesn't hurt to be explicit.
https://jestjs.io/docs/configuration#testenvironment-string

**Review by sophos-carlos — APPROVED** [review]
Looks good (trusting `dist/index.js` and `package-lock.json`)

File: package.json [hunk 2]
```diff
@@ -15,30 +15,27 @@
...
+    "@typescript-eslint/eslint-plugin": "^5.33.1",
+    "@typescript-eslint/parser": "^5.33.1",
+    "@vercel/ncc": "^0.34.0",
+    "eslint": "^8.22.0",
     "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-prettier": "^3.4.0",
+    "eslint-plugin-prettier": "^4.2.1",
     "husky": "^4.3.6",
-    "jest": "^27.2.5",
+    "jest": "^28.1.3",
```
Comment (bruce-bjorklund-sophos) [inline package.json:34]: @sophos-daniels yassssss

File: jest.config.ts [hunk 3]
```diff
@@ -5,7 +5,8 @@ const config: Config.InitialOptions = {
   testMatch: ['**/?(*.)+(test).+(ts|js)'],
   transform: {
     '^.+\\.ts$': 'ts-jest'
-  }
+  },
```
Comment (bruce-bjorklund-sophos) [inline jest.config.ts:8]: This makes me very happy

**Review by bruce-bjorklund-sophos — APPROVED** [review]

**Pull request merged**